sea-query-binders/src/sqlx_postgres.rs

```diff
@@ -0,0 +1,113 @@
+                }
+                Value::BigUnsigned(i) => {
+                    args.add(i.map(|i| <i64 as std::convert::TryFrom<u64>>::try_from(i).unwrap()));
+                }
+                Value::Float(f) => {
+                    args.add(f);
+                }
+                Value::Double(d) => {
+                    args.add(d);
+                }
+                Value::String(s) => {
+                    args.add(s.as_deref());
+                }
+                Value::Bytes(b) => {
+                    args.add(b.as_deref());
+                }
+                #[cfg(feature = "with-chrono")]
+                Value::ChronoDate(d) => {
+                    args.add(d.as_deref());
+                }
+                #[cfg(feature = "with-chrono")]
+                Value::ChronoTime(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-chrono")]
+                Value::ChronoDateTime(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-chrono")]
+                Value::ChronoDateTimeUtc(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-chrono")]
+                Value::ChronoDateTimeLocal(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-chrono")]
+                Value::ChronoDateTimeWithTimeZone(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-time")]
+                Value::TimeDate(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-time")]
+                Value::TimeTime(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-time")]
+                Value::TimeDateTime(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-time")]
+                Value::TimeDateTimeWithTimeZone(t) => {
+                    args.add(t.as_deref());
+                }
+                #[cfg(feature = "with-uuid")]
+                Value::Uuid(uuid) => {
+                    args.add(uuid.as_deref());
+                }
+                #[cfg(feature = "with-rust_decimal")]
+                Value::Decimal(d) => {
+                    args.add(d.as_deref());
+                }
+                #[cfg(feature = "with-bigdecimal")]
+                Value::BigDecimal(d) => {
+                    args.add(d.as_deref());
+                }
+                #[cfg(feature = "with-json")]
+                Value::Json(j) => {
+                    args.add(j.as_deref());
+                }
+                #[cfg(feature = "postgres-array")]
+                Value::Array(_) => {
+                    panic!("SeaQuery doesn't support array arguments for Postgresql");
```

Comment:
Why `SeaQuery`?) Maybe `sqlx`?

Reply:
So, `sqlx`  supports arrays for postgres, but the conversion is non-trivial.
Here, we would have to make sure that all the arguments are the same type, and convert them all to a static type before calling `args.add`.

Let's do that in a follow-up PR, maybe?

Reply:
Hmmm, I understand. Can you convert this comment to issues?

Reply:
Created https://github.com/SeaQL/sea-query/issues/336